exercises/acronym/src/example.c

```diff
@@ -0,0 +1,50 @@
+#include <ctype.h>
+#include <stdlib.h>
+
+static int count_words(const char phrase[]);
+static int is_first_letter_of_word(int c, int b);
```

Comment:
Ah, I normally work the other way, so that the file starts with an index of sorts of the static functions then the main functionality front and center and the static helper cruft comes after. Will adjust as you suggest though